src/components/Shared/NavFooter.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+  StyledNavFooter,
+  StyledNavFooterButtons,
+} from "../../styles/NavFooter.css.js";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHome,
+  faTruck,
+  faStreetView,
+  faWrench,
+} from "@fortawesome/free-solid-svg-icons";
+
+const navPages = [
+  {
+    title: "Home",
+    link: "/",
+    icon: faHome,
+  },
+  {
+    title: "My Work Orders",
+    link: "/my-work-orders",
+    icon: faStreetView,
+  },
+  {
+    title: "All Work Orders",
+    link: "/all-work-orders",
+    icon: faTruck,
+  },
+  {
+    title: "New Work Order",
+    link: "/work-order/new",
+    icon: faWrench,
```

Comment:
maybe this is a plus sign instead of a wrench?

Reply:
I tried it out, and it looks good with the plus sign! Maybe we should change all of the New Work Order icons to plus signs? Merging for now so we think about it.
![Screen Shot 2019-07-24 at 11 57 17 AM](https://user-images.githubusercontent.com/37249039/61812938-34c8ed80-ae0a-11e9-9990-5855b2ceb939.png)

Reply:
yeah I like that, I'll add it as an issue.